docs/_guide/styles.md

```diff
@@ -133,14 +174,23 @@ This can be a good way to load CSS generated from tools like SASS/LESS.
 There are some important caveats though:
 
 * External styles can cause a flash-of-unstyled-content (FOUC) while they load.
-* The URL in the `href` attribute is relative to the _main document_, making this technique mostly useful for application elements where asset URLs are well known, and not for reusable elements published publically.
+* The URL in the `href` attribute is relative to the _main document_, making this technique mostly useful for application elements where asset URLs are well known, and not for reusable elements published publicly.
 
+### What you can style
 
-## Styling the host element
+You can style:
+
+* [The host element](#host). The host element is your component. It is called the "host" because it hosts shadow DOM. You can style the element itself with special CSS selectors `:host` and `:host()`.
```

Comment:
"component": How consistently are we referring to "element" vs "component"? I've been trying to use "element" since we're always talking about HTML elements.